### PR TITLE
Acceptance tests for Sentry Nodes feature

### PR DESCRIPTION
**Pull request by lucassaldanha**
## PR Description

Running a remote validator using 3 beacon nodes, one per role. Verifies that each beacon node is being used for their designated purpose.

## Fixed Issue(s)
Related to #5929

### CODE REVIEW

File: acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuValidatorNode.java [hunk 1]
```diff
@@ -222,9 +268,34 @@ public TekuValidatorNode.Config withInteropValidators(
...
+
+      return this;
+    }
+
     public void writeConfigFile() throws Exception {
       final File configFile = File.createTempFile("config", ".yaml");
       configFile.deleteOnExit();
+
+      if (isUsingSentryNodeConfig) {
+        configMap.remove("beacon-node-api-endpoint");
```
Comment (rolfyone) [inline acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuValidatorNode.java:296]: does this reduce functionality available to the dsl? just wondering if future AT's won't understand why some api functions don't work, might be worth a log message...

Comment (lucassaldanha) [reply acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuValidatorNode.java:296]: Basically, one can't work with the other. If you are using sentry nodes, you cannot set a beacon node api endpoint. However, your duties provider sentry node will be used as your beacon node endpoint.

Comment (lucassaldanha) [reply acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuValidatorNode.java:296]: For reference: https://github.com/ConsenSys/teku/blob/master/teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorClientOptions.java#L105-L125

Comment (rolfyone) [reply acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuValidatorNode.java:296]: as long as dsl functions like getting data via rest api still work, I'm ok with that :)

Comment (lucassaldanha) [reply acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuValidatorNode.java:296]: It will work exactly the same way :)

**Review by rolfyone — APPROVED** [review]
LGTM

**Pull request merged**